app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
   domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
-  default from: "noreply@#{domain}"
+  default from: "CATja Screening <noreply@#{domain}>"
```

Comment:
Het is wel een productnaam, dus de spellingsregels zijn hier minder belangrijk imho.